lib/src/main/java/io/github/taz03/jia/responses/friendships/FriendshipsActionResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.github.taz03.jia.responses.friendships;
+
+import io.github.taz03.jia.responses.InstagramResponse;
+import io.github.taz03.jia.responses.models.friendship.Status;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FriendshipsActionResponse extends InstagramResponse {
+   @JsonProperty("friendship_status")
+    private Status friendshipStatus;
```

Comment:
i see that it needs more than just `Status` and also for the recently added show friendships request I'll be adding an `ExtendedStatus` model

Reply:
Done. U can now use `ExtendedStatus`

Reply:
i changed Status to ExtendedStatus. Should i change JsonProperty value?

Reply:
no, u r still parsing the same json value just as another format, it'll  stay the same